### PR TITLE
Document setting google assistant local discovery scan name and disabling http ssl

### PR DESCRIPTION
**Pull request by trvrnrth**
## Proposed change

Mirrors documentation describing the operation of the local fulfillment feature from https://www.nabucasa.com/config/google_assistant/.

Suggests use of the NGINX Home Assistant SSL proxy add-on to maintain secure external ingress.

Documents setting the local home SDK MDNS scan config Name to prevent IDENTIFY intents being sent for non-HA devices. Omitting this does not break the functionality, but results in a lot of logspam on every discovery round.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @trvrnrth, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21478).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @trvrnrth, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21478).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by calisro** [comment]
Just a suggestion.  

"Either wait for 30 minutes, or restart your connected Google device."  Should really be 

"Either wait for 30 minutes, or restart ALL your connected Google devices."

**Comment by frenck** [comment]
@trvrnrth We had multiple PRs that hit kinda the same thing, some have been merged, others have been closed.

However, your PR is quite extensive. Could you maybe check the current state of the page against this PR and see if you think we still need adjustments?

**Comment by trvrnrth** [comment]
@frenck The main addition of adding the scan config name has been pulled in elsewhere now, but there are some minor typos vs the actual names in the console which I've corrected.

I think it's probably also worth pulling in a bit of the overview (as included at https://www.nabucasa.com/config/google_assistant/) so I've taken the opportunity to distill that down a bit and keep the note format for the SSL option used in one of the other merged PRs.

File: source/_integrations/google_assistant.markdown [hunk 1]
```diff
@@ -109,30 +109,42 @@ If you want to support active reporting of state to Google's server (configurati
...
+Google Assistant devices can send their commands locally to Home Assistant allowing them to respond faster.
+
+Your Home Assistant instance needs to be connected to the same network as the Google Assistant device that you’re talking to so that it can be discovered via mDNS discovery (UDP broadcasts).
 
-The [`ssl_certificate` option](/integrations/http/#ssl_certificate) in the `http` integration must not have a value or this feature won't work.
+Your Google Assistant devices will still communicate via the internet to:
+- sync entities
+- get credentials to establish a local connection
+- send commands that involve a [secure device](#secure-device)
+- send commands if local fulfillment fails
```
Comment (frenck) [inline source/_integrations/google_assistant.markdown]:
```suggestion
- sync entities
- Get credentials to establish a local connection.
- Send commands that involve a [secure device](#secure-device).
- Send commands if local fulfillment fails.
```

File: source/_integrations/google_assistant.markdown [hunk 2]
```diff
@@ -109,30 +109,42 @@ If you want to support active reporting of state to Google's server (configurati
...
 ### Enable Local Fulfillment
 
-<div class='note'>
+Google Assistant devices can send their commands locally to Home Assistant allowing them to respond faster.
+
+Your Home Assistant instance needs to be connected to the same network as the Google Assistant device that you’re talking to so that it can be discovered via mDNS discovery (UDP broadcasts).
 
-The [`ssl_certificate` option](/integrations/http/#ssl_certificate) in the `http` integration must not have a value or this feature won't work.
+Your Google Assistant devices will still communicate via the internet to:
+- sync entities
```
Comment (trvrnrth) [inline source/_integrations/google_assistant.markdown]:
```suggestion
- Sync entities.
```

Comment (trvrnrth) [reply source/_integrations/google_assistant.markdown]: @frenck Presume you meant to do this one as well :)

**Review by frenck — APPROVED** [review]
Thanks, @trvrnrth 👍

**Pull request merged**